src/fragments/lib/auth/native_common/signin_next_steps/common.mdx

```diff
@@ -2,40 +2,70 @@ After a user has finished signup, they can proceed to sign in. Amplify Auth sign
 
 import ios0 from "/src/fragments/lib/auth/ios/signin_next_steps/10_signin.mdx";
 
+import android1 from "/src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx";
+
 <Fragments fragments={{ios: ios0}} />
 
+<Fragments fragments={{android: android1}} />
+
 ### Confirm signin with SMS MFA
 
-import ios1 from "/src/fragments/lib/auth/ios/signin_next_steps/20_confirm_sms_mfa.mdx";
+import ios2 from "/src/fragments/lib/auth/ios/signin_next_steps/20_confirm_sms_mfa.mdx";
```

Comment:
nit: Do we need to change ios vars here?

Reply:
Nope. I was under the impression everything needs to go in linear order. That is not true. Updated and resolved.